app/models/inventory.rb

```diff
@@ -1,13 +1,13 @@
 # creating Inventory model for all User's stuff
 class Inventory < ApplicationRecord
-  # belongs_to :user
-  # belongs_to :item
+  belongs_to :user
+  belongs_to :item
   validates_uniqueness_of :item_id, scope: :user_id
   validate :maximum_items_not_more_then_15, on: :create
 
   private
 
   def maximum_items_not_more_then_15
-    errors.add(:base, 'Your backpack is full!') if user_id.count > 15
+    errors.add(:base, 'Your backpack is full!') if user.items.count > 15
```

Comment:
1. Please forget about the magic numbers. Use constant instead. And rename method to something more simple and understandable. f.e. `check_items_limit` or just `items_limit`.
2. Think about using `counter_cache` if you often need items count.